src/bp-groups/bp-groups-template.php

```diff
@@ -420,7 +423,7 @@ function bp_has_groups( $args = '' ) {
 			'max'                => false,
 			'show_hidden'        => false,
 			'page_arg'           => 'grpage',
-			'user_id'            => bp_displayed_user_id(),
+			'user_id'            => bbp_get_current_user_id(),
```

Comment:
`bbp_get_current_user_id` function is a part of the forums and when any site have a forum component is disabled then it will trigger a Fatal error and site will be crashed.